src/Tizen.NUI/src/public/BaseComponents/ImageView.cs

```diff
@@ -793,6 +803,41 @@ private FittingModeType InternalFittingMode
             }
         }
 
+        /// <summary>
+        /// Returns the natural size of the view.
+        /// </summary>
+        /// <remarks>
+        /// Deriving classes stipulate the natural size and by default a view has a zero natural size.
+        /// </remarks>
+        /// <since_tizen> 5 </since_tizen>
```

Comment:
I think it's copy-paste error.
Could you remove since_tizen tag at 812 line and 830 line?

Reply:
This is intention. `NaturalSize` and `NaturalSize2D` API opened since that version. and ImageView's `NaturalSize` is just call base class function.
I don't want to doing additional (and USELESS) ACR process about it.
And also want to make users can notify this API's informations.

Reply:
More information. `UpdateImage` call is just our internal process things that APP developer don't need to know the difference ; with `UpdateImage` call versers without `UpdateImage` call. That's why I just copy API version information.

Reply:
Hmm I understood what you mean.
But, these properties are new and public ones in ImageView class. Only View class did ACR process for them. It depends on Platform rules.
Actually, the tags won't be a big problem because of `[EditorBrowsable(EditorBrowsableState.Never)]` tag below. However, it would be a little confusing for someone who doesn't know the history in the future.
I think it would be better to check them as Note, etc.

Let's ask NUI other guys.
@dongsug-song , @taehyub , @jaehyun0cho , @everLEEst   What do you think? We need to rule on this for the future.

Reply:
I checked other class's `public new` usage. most of them are ACR'ed. But `AnimatedImageView.cs` 's `ResourceUrl` or `Play` functions didnt ACR so just setup as hidden even they did exactly same job with `ImageView`. So I just follow up that case, and remove `since` tag.